.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on:
+  push:
+    branches:
+      - "*"
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.7'
+    - name: Install dependencies
+      run: |
+        python setup.py sdist
```

Comment:
I think the Travis CI code was using `pytest` to run the tests and flake8 for linting, any chance you'd be able to hook those up too? Looks like I had it hooked up to Codecov for coverage reporting, not sure if it's worthwhile trying to get that working but wondering if Github just has something that's super cheap/quick to set up?

Reply:
created this as an issue here: https://github.com/zillow/battenberg/issues/33

I definitely want to deploy this and unblock the issue Anton resolved first and worry about testing/linting in another PR